corehq/apps/hqmedia/models.py

```diff
@@ -794,7 +794,8 @@ def multimedia_map_for_build(self, build_profile=None):
         if not build_profile or not domain_has_privilege(self.domain, privileges.BUILD_PROFILES):
             return self.multimedia_map
 
-        requested_media = self.logo_paths   # logos aren't language-specific
+        requested_media = set()
+        requested_media |= self.logo_paths   # logos aren't language-specific
```

Comment:
@orangejenny can you add a comment why we are doing this?

Reply:
Sets are assigned by reference, not by value, so I think (and please correct me if I'm wrong @orangejenny) the problem is that two lines below this,
```python
    requested_media |= self.all_media_paths(lang=lang)
```
affected the value of `self.logo_paths`. This change makes `requested_media` a copy of `self.logo_paths`.

Alternative ways of writing this could be
```python
    requested_media = copy(self.logo_paths)
```
or
```python
    requested_media = set(self.logo_paths)
```
but I'm happy with this as it is. ... although :thinking: maybe using `copy()` would make a comment unnecessary.

Reply:
Thanks @kaapstorm

Reply:
Thanks @kaapstorm 
I agree `copy()` would be clearer, updated in bb4162509d27315b0aa50679ec9082a696e0cceb